src/components/TokenBrowser.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import jwtDecode from 'jwt-decode';
+import { getClientConfig, JWTPayload } from '../clients';
+import { useClient } from '../clients/client';
+import styles from './styles.module.css';
+
+const TokenBrowser = (): React.ReactElement => {
+  const client = useClient();
+  const [selectedToken, changeSelectedToken] = useState<string | undefined>();
+  const [selectedId, changeSelectedId] = useState<string | undefined>();
+  const apiTokens = client.getApiTokens();
+  const userTokens = client.getUserTokens();
+  const config = getClientConfig();
+  const isToken =
+    selectedId &&
+    selectedToken &&
+    selectedToken.length > 20 &&
```

Comment:
Where does the `20` come from? Can this magic number be given a name, so that it's clearer what it signifies?

Reply:
Named the prop and added comment:
// Keycloak server returns object with also other data than tokens.
  // 20 is a (pretty) safe length of non-token property value.
  // Later on tokens may be filted with prop names and config.audience.
  // At the moment keycloak server seems to change quite often, so this simple solution is used